Docs/Configuration.tex

```diff
@@ -4422,6 +4413,120 @@ \subsection{Security Properties}\label{miscsecurityprops}
 
 \end{enumerate}
 
+\subsection{Serial Properties}\label{miscserialprops}
+
+\begin{enumerate}
+\item
+  \texttt{BaudRate}\\
+  \textbf{Type}: \texttt{plist\ integer}\\
+  \textbf{Failsafe}: \texttt{115200}\\
+  \textbf{Description}: Set the baud rate for serial port.
+
+  This option will override the value of \texttt{gEfiMdeModulePkgTokenSpaceGuid.PcdSerialBaudRate}
+  defined in \href{https://github.com/acidanthera/audk/blob/master/MdeModulePkg/MdeModulePkg.dec}{MdeModulePkg.dec}.
+
+\item
+  \texttt{ClockRate}\\
+  \textbf{Type}: \texttt{plist\ integer}\\
+  \textbf{Failsafe}: \texttt{1843200}\\
+  \textbf{Description}: Set the clock rate for serial port.
+
+  This option will override the value of \texttt{gEfiMdeModulePkgTokenSpaceGuid.PcdSerialClockRate}
+  defined in \href{https://github.com/acidanthera/audk/blob/master/MdeModulePkg/MdeModulePkg.dec}{MdeModulePkg.dec}.
+
+\item
+  \texttt{ExtendedTxFifoSize}\\
+  \textbf{Type}: \texttt{plist\ integer}\\
+  \textbf{Failsafe}: \texttt{64}\\
+  \textbf{Description}: Set the extended transmit FIFO size for serial port.
+
+  This option will override the value of \texttt{gEfiMdeModulePkgTokenSpaceGuid.PcdSerialExtendedTxFifoSize}
+  defined in \href{https://github.com/acidanthera/audk/blob/master/MdeModulePkg/MdeModulePkg.dec}{MdeModulePkg.dec}.
+
+\item
+  \texttt{FifoControl}\\
+  \textbf{Type}: \texttt{plist\ integer}\\
+  \textbf{Failsafe}: \texttt{0x07}\\
+  \textbf{Description}: Configure serial port FIFO Control settings.
+
+  This option will override the value of \texttt{gEfiMdeModulePkgTokenSpaceGuid.PcdSerialFifoControl}
+  defined in \href{https://github.com/acidanthera/audk/blob/master/MdeModulePkg/MdeModulePkg.dec}{MdeModulePkg.dec}.
+
+\item
+  \texttt{Init}\\
+  \textbf{Type}: \texttt{plist\ boolean}\\
+  \textbf{Failsafe}: \texttt{false}\\
+  \textbf{Description}: Perform serial port initialisation.
+
+  This option will perform serial port initialisation within OpenCore prior to enabling
```

Comment:
You forgot to mention that all the other `Serial` fields are not used and left as failsafe values when `Init` is `false`.

Reply:
https://github.com/acidanthera/OpenCorePkg/commit/5aaf2a06b605a536545c62bce41b22c633ececc3